test/runtests.jl

```diff
@@ -1,8 +1,9 @@
 # This file is a part of LegendTextIO.jl, licensed under the MIT License (MIT).
 
-import Test
-Test.@testset "Package LegendTextIO" begin
+using Test, LegendTextIO, Tables
 
-# test code goes here
+@testset "LegendTextIO.jl" begin
```

Comment:
Please use `Test.@testset "Package LegendTextIO" begin`

Reply:
This is ultimately a taste thing, so sure. I am curious though. Why? I think my version is the more common style and simpler, no?

Do you want `Test.@testset` for every testset by the way?

Reply:
>  I am curious though. Why?

Reply:
> I am curious though. Why?

Just because I/we've been using it usually so far. But you're right, it doesn't matter much.

> Do you want Test.@testset for every testset by the way